lib/Grape.js

```diff
@@ -32,32 +56,38 @@ class Grape extends Events {
       maxSize: 5000,
       maxAge: Math.ceil(this.conf.dht_peer_maxAge / 2)
     })
-
+    this._initError = null
     this._interface = {}
     this._active = false
     this.listening = false
+    this[kOnWarning] = (...args) => {
+      debug(this.conf.dht_port, 'warning', ...args)
+      this.emit('warning', ...args)
+    }
   }
 
   createNode (cb) {
     const dht = this.dht = DHT({
-      // maxTables: this.conf.dht_maxTables,
-      // maxValues: this.conf.dht_maxValues,
-      bootstrap: this.conf.dht_bootstrap || undefined,
+      maxValues: this.conf.dht_maxValues,
+      bootstrap: this.conf.dht_bootstrap,
       // timeBucketOutdated: this.conf.dht_nodeLiveness,
```

Comment:
do we need these? why is it commented. i am confused (same for above values)

Reply:
oh yeah, I commented these out a long time ago so that I could move on with the major implementation pieces and then trigger a discussion at a later stage (which is now :D)

`maxValues` comment can be removed (because it's actually already there on line 71)
`maxTables` is only relevant to bt dht, we don't use that abstraction in hs dht so we can remove it 

`timeBucketOutdated` is the one I'm unsure about, I need to check that it's the same as our `maxAge` in hs dht

Reply:
so `timeBucketOutdated` in bt dht is essentially a ping interval which is the equivalent of the tick interval in dht-rpc - it's hardcoded at 5s (https://github.com/mafintosh/dht-rpc/blob/master/index.js#L48) - I think the philosophy around this is there's an optimal interval pinging nodes which the dht itself should be responsible for setting. Is this a configuration you rely on or use frequently?

Reply:
removed :)